src/main/java/nextstep/subway/line/domain/Section.java

```diff
@@ -87,8 +87,8 @@ public void updateForDelete(Section section) {
         this.distance += section.distance;
     }
 
-    public boolean isDummy() {
-        return this == DUMMY_SECTION;
+    public boolean isEmpty() {
```

Comment:
요청사항 반영 좋습니다 👍 

그리고 이왕 잘 해주셨으니 좀 더 참고할만한 부분 공유 드리면
`Collections.emptyList()`가 제공 되고 있습니다 
해당 코드도 한번 참고해보시면 좋을 것 같습니다 😄 

```java
    public static final List EMPTY_LIST = new EmptyList<>();

    /**
     * Returns an empty list (immutable).  This list is serializable.
     *
     * <p>This example illustrates the type-safe way to obtain an empty list:
     * <pre>
     *     List&lt;String&gt; s = Collections.emptyList();
     * </pre>
     *
     * @implNote
     * Implementations of this method need not create a separate {@code List}
     * object for each call.   Using this method is likely to have comparable
     * cost to using the like-named field.  (Unlike this method, the field does
     * not provide type safety.)
     *
     * @param <T> type of elements, if there were any, in the list
     * @return an empty immutable list
     *
     * @see #EMPTY_LIST
     * @since 1.5
     */
    @SuppressWarnings("unchecked")
    public static final <T> List<T> emptyList() {
        return (List<T>) EMPTY_LIST;
    }

```

Reply:
get(), add()가 안되는 싱글턴 객체군요!
비어있는 리스트를 사용할 때, 사용하도록 하겠습니다.
공유 감사합니다!